src/components/shared/scheduler/SchedulerEditor.vue

```diff
@@ -308,7 +153,7 @@
 
                 schedulerObject['service=scheduler;table=timetables;field=starts_at;'] = schedule.limit.startsAt ? moment(schedule.limit.startsAt).format('YYYY-MM-DD hh:mm:ss') : null
                 schedulerObject['service=scheduler;table=timetables;field=ends_at;'] = schedule.limit.endsAt ? moment(schedule.limit.endsAt).format('YYYY-MM-DD hh:mm:ss') : null
-                schedulerObject['service=scheduler;table=timetables;field=max_executions;'] = schedule.limit.maxExecutions > 0 ? parseInt(schedule.limit.maxExecutions) : null
+                schedulerObject['service=scheduler;table=timetables;field=max_executions;'] = schedule.limit.maxExecutions > 0 ? parseInt(schedule.limit.maxExecutions, 10) : null
```

Comment:
Does this apply a default of 10 maximum executions if `maxExecutions` cannot be parsed as an integer? If so, should we possibly apply `null` here as the default?

Reply:
No this is the 'base' for the integer to be displayed in, base 10 is the 'decimal' numbering system. (As opposed to binary for example)

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt

This was actually picked up by my linter, not sure how it got this far already.

Reply:
Ah ok, cool stuff!

Reply:
For simple number casting to in JS, I tend to use `Number('12')`